### PR TITLE
Use tee to write to /etc/bash_completion.d/kubectl

### PR DESCRIPTION
**Pull request by CarlosDomingues**
The current instructions will probably result in "Permission denied" for most folks.

### CODE REVIEW

File: content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ You now need to ensure that the kubectl completion script gets sourced in all yo
 - Add the completion script to the `/etc/bash_completion.d` directory:
 
    ```bash
-   kubectl completion bash >/etc/bash_completion.d/kubectl
+   echo -e "$(kubectl completion bash)" | sudo tee /etc/bash_completion.d/kubectl > /dev/null
```
Comment (sftim) [inline content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:40]: This is better:
```suggestion
   kubectl completion bash | sudo tee /etc/bash_completion.d/kubectl > /dev/null
```




**Ideally**: use https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs to cover the two cases:
1. per user: `kubectl completion bash >>~/.bashrc`
2. system-wide change, editing `/etc/bash_completion.d/kubectl`

and also mention that this is specific to bash; other shells exist.

**Review by CarlosDomingues — COMMENTED** [review]
Ok that makes sense!

**Comment by jimangel** [comment]
Congrats on your first PR to k/website @CarlosDomingues! 🎉 

I understand the reasoning behind the change, I'm not sure I understand the distinction of tabs being "System" vs. "User" or when either of those tabs would apply to me (as a cluster operator). Is there a better way to clarify the target audience or when each section is applicable?

Preview link: https://deploy-preview-30662--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/included/optional-kubectl-configs-bash-linux/

Lastly, when time permits will you please [squash your commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits)?

Thanks!

**Comment by sftim** [comment]
> the distinction of tabs being "System" vs. "User" 

The two cases are, as I understand it:
- installing `kubectl` on your client PC, just for you (and not using superuser access)
- installing `kubectl` on a client system, perhaps a bastion or “jump box”,  so that the tool is available for all users (requires superuser access)

**Comment by annajung** [comment]
The use of "User" and "System" seems okay to me
/lgtm

**Review by sftim — COMMENTED** [review]
I'd be happy to see a follow-up PR that puts a comment in each tab to clarify the circumstances when you'd pick each option.

/approve

**Pull request merged**